config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/cloud-bulldozer/benchmark-operator
+  newName: quay.io/vchalla/benchmark-operator
```

Comment:
If testing is completed please revert this.

Reply:
Done